jose-jwt/jwe/JWE.cs

```diff
@@ -0,0 +1,330 @@
+﻿#if NETSTANDARD
+namespace Jose.jwe
+{
+    using Newtonsoft.Json;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    public class JweRecipient

```

Comment:
and this guy to `Recipient`. 

So outside it all become: 
```c#
using Jose;

new Jwe.Recipient(..);
```

?

Reply:
Now done